apps/boilerplate-front-end/package.json

```diff
@@ -9,8 +9,8 @@
     }
   },
   "scripts": {
-    "start": "vite --port=5173",
-    "dev": "vite --port=5173",
+    "start": "vite --port=3000",
```

Comment:
Why don't let 5173 because when we'll change okta login for true we'll forget to change this port